packages/circuits/test/initial.test.js

```diff
@@ -21,8 +21,8 @@ describe("Initial Hand Test", () => {
     });
 
     // set longer timeout for test
-    jest.setTimeout(10000);
-    it("Should correctly construct a merkle tree root", async () => {
+    jest.setTimeout(25000);
```

Comment:
Increasing the timeout because running more tests in parallel make each test slower too. Not sure if jest can run tests sequentially...